grpc/errors_test.go

```diff
@@ -0,0 +1,57 @@
+package grpc
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"github.com/letsencrypt/boulder/core"
+	testproto "github.com/letsencrypt/boulder/grpc/test_proto"
+	"github.com/letsencrypt/boulder/probs"
+	"github.com/letsencrypt/boulder/test"
+)
+
+type errorServer struct{}
+
+func (s *errorServer) Chill(ctx context.Context, in *testproto.Time) (*testproto.Time, error) {
+	var err error
+	switch *in.Time {
```

Comment:
Note that this would require moving the server setup inside the testCases loop, which I think is fine.